aws-rds-integration/src/main/java/software/amazon/rds/integration/IntegrationStatusUtil.java

```diff
@@ -0,0 +1,40 @@
+package software.amazon.rds.integration;
+
+import com.google.common.collect.ImmutableSet;
+import software.amazon.awssdk.services.rds.model.IntegrationStatus;
+
+import java.util.Set;
+
+public class IntegrationStatusUtil {
+
+    //     CREATING("creating"),
+    //    ACTIVE("active"),
+    //    MODIFYING("modifying"),
+    //    FAILED("failed"),
+    //    DELETING("deleting"),
+    //    SYNCING("syncing"),
+    //    NEEDS_ATTENTION("needs_attention"),
+    //    UNKNOWN_TO_SDK_VERSION((String)null);
+    private final Set<IntegrationStatus> VALID_CREATING_STATES = ImmutableSet.of(
+            IntegrationStatus.CREATING,
+            IntegrationStatus.SYNCING,
+            IntegrationStatus.MODIFYING,
+            IntegrationStatus.NEEDS_ATTENTION,
+            IntegrationStatus.ACTIVE
```

Comment:
The code asserts only the valid states, so I think it will work correctly (I flipped the direction from the older revision, maybe that caused the confusion).

I added another test about validating all known statuses, but I am not sure if that's OK. Let me know what you think.